src/status_im/contexts/profile/settings/events.cljs

```diff
@@ -91,6 +91,18 @@
              (boolean private?)
              {})))
 
+(rf/defn toggle-peer-syncing
+  {:events [:profile.settings/toggle-peer-syncing]}
+  [{:keys [db] :as cofx}]
+  (let [value (get-in db [:profile/profile :peer-syncing-enabled?])
+        new-value (not value)]
+    (rf/merge cofx
+              {:json-rpc/call [{:method     "wakuext_togglePeerSyncing"
+                                :params     [new-value]
+                                :on-success #(log/info "successfully toggled peer syncing" new-value)
```

Comment:
I guess we can skip logging on success unless it's strictly required, as logging is one of the performance bottlenecks.

Reply:
I didn't know it was a perf problem. @smohamedjavid, do you have more information to share about the specifics that led to this conclusion?

Reply:
https://reactnative.dev/docs/performance#using-consolelog-statements

Reply:
oh nice, that's not something as well I was aware, this case doesn't seem problematic as it's not perfomance sensitive, but I guess we should strip them out as well?